src/data_management/rasters/Raster.js

```diff
@@ -352,6 +352,7 @@ class Raster extends Component {
                       </NavLink>
                     ) : // if raster.data.name contains "Optimizer OR "RasterStoreSource" then there is already data in the raster and the user is also allowed to update this
                      (!raster.is_geoblock &&
+                      raster.source && raster.spatial_bounds &&
                       (raster.source.name.split("_")[0] === "Optimizer" ||
```

Comment:
Geen enkele `source.name` zal beginnen met `Optimizer` of `RasterStoreSource`. Deze logica zorgt er nu voor dat er geen data geupload kan worden.